benches/benches/helpers.rs

```diff
@@ -1,18 +1,16 @@
 use futures::channel::oneshot;
-use jsonrpsee_http_server::{HttpConfig, HttpServer};
+use jsonrpsee_http_server::HttpServerBuilder;
 use jsonrpsee_ws_server::WsServer;
 
 /// Run jsonrpsee HTTP server for benchmarks.
 pub async fn http_server() -> String {
 	let (server_started_tx, server_started_rx) = oneshot::channel();
 	tokio::spawn(async move {
-		let server = HttpServer::new("127.0.0.1:0", HttpConfig { max_request_body_size: u32::MAX }).await.unwrap();
-		let mut say_hello = server.register_method("say_hello".to_string()).unwrap();
-		server_started_tx.send(*server.local_addr()).unwrap();
-		loop {
-			let r = say_hello.next().await;
-			r.respond(Ok("lo".into())).await.unwrap();
-		}
+		let mut server =
+			HttpServerBuilder::default().max_request_body_size(u32::MAX).build("127.0.0.1:0".parse().unwrap()).unwrap();
```

Comment:
It's a way to make the system to find a suitable port number instead of doing it manually as we spawn a bunch of servers in parallel for some tests. In other words, a `wildcard port`

I guess we could spawn just a server and use it for all tests but works fine so far.

Reply:
nice, TIL. I think it's fine the way it is